scripts/config_dealii.sh

```diff
@@ -30,6 +30,12 @@ DEAL=$WORKING_DIRECTORY/sw/dealii
 
 rm -rf CMakeFiles/ CMakeCache.txt
 
+# Note on compiler flags: Note that "-march=native" requires that the hardware 
+# on which you compile the code is consistent with the hardware on which you 
+# execute the code. If this is not the case, consider to specify the target 
+# hardware for compilation, e.g. "-march=haswell" or "-march=skylake-avx512", 
+# where "-march=haswell" can also be used for recent AMD x86-64 hardware. 
```

Comment:
There are also options for amd processors like `-march=znver2` for Zen 2 architecture. 

https://gcc.gnu.org/onlinedocs/gcc/x86-Options.html

Reply:
thanks for this hint. I adjusted the comment with a reference to the link you have provided.

Reply:
That's indeed an excellent link. Just as a background information, in my experience the exact microarchitecture (e.g., znver2 versus haswell) has only a minor influence on the performance - it affects some minor scheduling issues or preferring one instruction over another, but all of that are handled relatively well in hardware, too. The big influence is which extensions get available, such as FMA, AVX/AVX2, AVX-512, etc., that can map the code e.g. to wider SIMD registers or more powerful instructions, which are not specific to single CPU models like Haswell.

Reply:
thanks for the additional information.